finance/pay.go

```diff
@@ -61,6 +61,15 @@ func (f *Finance) Pay(fnid string) (errs []string) {
 			continue
 		}
 
+		// get token tag
+		tokenTag, ok := f.tokenTagMap[finData.TargetToken]
+		if !ok {
+			msg := fmt.Sprintf("Target token not exist: %s ; nid/id: %v/%v", finData.TargetToken, fnid, page.ID)
+			log.Error(msg)
```

Comment:
log.Error("Target token not exist", "targetToken", finData.TargetToken, "fnid", fnid, "page", page.ID)